enlighten/StatusIndicators.py

```diff
@@ -123,7 +123,8 @@ def update_visibility(self):
             # Light Source (Lamp, Laser etc)
             ####################################################################
 
-            if settings.eeprom.has_laser:
+            all_specs = self.multispec.get_spectrometers()
+            if settings.eeprom.has_laser and len(all_specs) <= 1:
```

Comment:
Just an FYI...this is all inside a "spec is not None" so I don't think it's possible for len() to be <1 at this point.  Code's fine though.